bin/serial-number

```diff
@@ -7,11 +7,17 @@ var fs = require('fs');
 
 var serialNumber = require('../');
 
+var cmdprefix = '/usr/sbin/';
 if (process.argv[process.argv.length - 1] === '--uuid') {
 	serialNumber.preferUUID = true;
+    if (process.argv[process.argv.length - 2].indexOf('--cmdprefix') > -1) {
```

Comment:
This needs to be refactored to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Also, introducing a new CLI switch means the original simple implementation isn't sufficient anymore, such as if both options are used with `--cmdprefix` being specified first.

There are modules on npm that handle this, but I'd prefer a simple implementation without extra dependencies if possible. Untested, but perhaps something like the following would work:

```js
process.argv.forEach(function(arg, index, arr) {
  if (arg === '--uuid') {
    serialNumber.preferUUID = true;
  } else if (arg === '--cmdprefix') {
    cmdPrefix = arr[index + 1];
  }
}
```